apps/ledger-live-mobile/src/screens/Nft/NftGallery/ReceiveNFTsModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { BottomDrawer, Button, Text } from "@ledgerhq/native-ui";
+
+import { NavigatorName, ScreenName } from "../../../const";
+import { track, TrackScreen } from "../../../analytics";
+
+type Props = {
+  navigation: any;
```

Comment:
no any please, we are banning "any" soon :smile: 
Either find the right type, of just use "useNavigation" to get navigation in the modal component